src/webui/www/private/views/preferences.html

```diff
@@ -122,6 +122,12 @@
                     <input type="text" id="exportdirfin_text" autocorrect="off" autocapitalize="none" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <input type="checkbox" id="multirename_rememberprefs_checkbox" onchange="LocalPreferences.set('multirename_rememberPreferences', this.checked);" />
+                    <label for="multirename_rememberprefs_checkbox">QBT_TR(Remember Multi-Rename Preferences)QBT_TR[CONTEXT=OptionsDialog]</label>
```

Comment:
About capitalization, not sure if "Multi-Rename" is special but "Preferences" is an ordinary word and other settings use lower case
```suggestion
                    <label for="multirename_rememberprefs_checkbox">QBT_TR(Remember Multi-Rename preferences)QBT_TR[CONTEXT=OptionsDialog]</label>
```

/Off topic for this line
You should disable/remove "Rename files..." entry in torrent list's context menu when torrent doesn't have metadata and when multiple torrents are selected.

Other than that LGTM.

Reply:
Fixed the capitalization and added a check that will hide `Set Location...`, `Rename...`, and `Rename Files...` when multiple torrents are selected

Reply:
Why `Set Location`?

Reply:
When multiple torrents were selected, the `Select Location...` menu item had no effect

Reply:
Seems you took it incorrectly. It just extracts current save path from the first selected torrent. But it send all the selected torrents hashes:
https://github.com/loligans/qBittorrent/blob/0939417f1b225758f24df49867a7ca657fc0156c/src/webui/www/private/scripts/mocha-init.js#L492

Reply:
@loligans 
General comment is making unrelated changes is the best way to slow down your PR.

Reply:
my mistake, i'll undo that change.

Reply:
The change has been reverted, now the only menu item's that get hidden are `Rename...` and `Rename Files...` when multiple torrents are selected

Reply:
> when multiple torrents are selected

What about the case when single torrent is selected but it has no metadata?

Reply:
i can make that change too, although i'm not sure where i can check that a torrent has metadata. 

i'm not familiar with torrent metadata and so i could use a little help with knowing where to look on the webui

Reply:
> i can make that change too, although i'm not sure where i can check that a torrent has metadata.

It is not explicitly represented via WebAPI. However, since a torrent without metadata doesn't know anything about its payload, you can estimate some kind of indirect field, such as `total_size` which is `-1` in case when there is no metadata available.
Or just use the way suggested by @thalieht in https://github.com/qbittorrent/qBittorrent/pull/18287#discussion_r1103803194.

Reply:
thanks! i'll look at both options and add the functionality.